adafruit_si4713.py

```diff
@@ -343,6 +363,46 @@ def tx_antenna_capacitance(self):
         # (scaled appropriately for pF units).
         return self._BUFFER[6] * 0.25
 
+    @property
+    def digital_input_sample_rate(self):
+        """Read the digital input sample rate. This property may
+        only be set or read when in powerup mode. TX_TUNE_FREQ
+        command must be sent after the POWER_UP command to start
+        the internal clocking before setting this property.
+        """
+        print("test DISR, no value, not yet implemented")
+        # TODO
+
+    @digital_input_sample_rate.setter
+    def digital_input_sample_rate(self, val):
+        print("test DISR, with value")
+        assert (val == 0) or (32000 <= val <= 48000)
+        print("will set", val)
+        if self._set_property(_SI4713_PROP_DIGITAL_INPUT_SAMPLE_RATE, val):
+            print("have set, return true")
+        else:
+            print("have set, return false")
```

Comment:
same as other prints, suggest removing or adding a `verbose` argument to `__init__()` that enables the printing